Added stopVideoRecording callback in config (#87)
fluttercandies/flutter_wechat_camera_picker: README.md

| previewTransformBuilder      | `PreviewTransformBuilder?`  | The widget builder which will transform the camera preview.                                           | null                                   |
| onEntitySaving               | `EntitySaveCallback?`       | The callback type define for saving entity in the viewer.                                             | null                                   |
| onError                      | `CameraErrorHandler?`       | The error handler when any error occurred during the picking process.                                 | null                                   |

> | onXFileCaptured              | `XFileCapturedCallback?`    | The callback type define for capturing xfile in the camera.                                           | null                                   |

Reviewer: ```suggestion
| onXFileCaptured              | `XFileCapturedCallback?`    | The callback type defintion when the XFile is captured by the camera.                                     | null                                   |
```

> | onStopRecordingVideo                      | `XFile?`       | The callback type define for customizing the behaviour after video recording stops. Once provided, the default video viewer page will not be pushed after on video stop recording.                                  | null                                   |

Reviewer: Same as above.

Author: Aligned.
